docs/design/third-party-libraries.md

```diff
@@ -0,0 +1,175 @@
+# Installing third party libraries
+
+# Abstract
+
+The current `ng install` process is faulty; third parties have to add a `bundles/` directory
+with metadata that we expect. This document explores ways to improve on the process.
+
+# Requirements
+
+The following use cases need to be supported by `ng install`:
+
+1. Support for _any_ thirdparties, ultimately falling back to running `npm install` only and
+doing nothing else, if necessary.
+1. Not a new package manager.
+1. Metadata storage by thirdparties in `package.json`. This must be accessible by plugins for
+the build process or even when scaffolding.
+1. Proper configuration of SystemJs in the browser.
+1. SystemJs configuration needs to be kept separated from the autogenerated part.
+1. The build process right now is faulty because the `tmp/` directory used by Broccoli is
+inside the project, which means that `node_modules/` is understood by Typescript. We need to
+proper map the imports and move them to `dist/`.
```

Comment:
I don't understand what you mean here at all, but might just be my fault. Can you dumb it down a bit for me?